packages/editor-ui/src/components/InputPanel.vue

```diff
@@ -118,7 +118,7 @@ export default mixins(
 			return window.localStorage.getItem(LOCAL_STORAGE_MAPPING_FLAG) === 'true';
 		},
 		showDraggableHint(): boolean {
-			const toIgnore = [START_NODE_TYPE, CRON_NODE_TYPE, INTERVAL_NODE_TYPE];
+			const toIgnore = [START_NODE_TYPE, MANUAL_TRIGGER_NODE_TYPE, CRON_NODE_TYPE, INTERVAL_NODE_TYPE];
```

Comment:
I think you missed adding `SCHEDULE_TRIGGER_NODE_TYPE` here

Reply:
it's not needed here.. because unlike the cron node, it actually returns output on manual execution.... really this should be changed to depend if input panel has output.. will change it to that

Reply:
I see, then I guess we can remove the `SCHEDULE_TRIGGER_NODE_TYPE` import above

Reply:
ops saw the comment about removing the import too late..